CMS-Enterprise/mint-app
# [MINT-3224]mto common solution pocs

# MINT-3224

## Description

This migrates existing pocs to be used for mto. It takes all existing functionality and makes it applicable to MTO, including a resolver to get a primary POC.

You'll notice instead of a separate resolver, the data type is nested in a wrapper type, MTOCommonSolutionContactInformation so we can get the poc without needing to build another data loader.


## How to test this change

Using Postman
1.  Make a model Plan
2. Query all solutions with point of contact information
3. Verify the information returns correctly
4. Verify you can also return the primary point of contact easily

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
